src/platform/forms-system/src/js/validation.js

```diff
@@ -473,6 +473,23 @@ export function validateDateRange(
   }
 }
 
+export function validateDateRangeAllowSameMonth(
```

Comment:
Since this function is identical to `validateDateRange` with the exception of the boolean argument in `isValidDateRange`, can you refactor both functions so that this acts as a helper that passes a `true` value for that call? Hopefully that makes sense :sweat_smile:

Reply:
Updated. I'm taking over Nick's PRs - he's on the Lighthouse team now.